src/Microsoft.Health.Dicom.Azure/KeyVaultSecretClientOptions.cs

```diff
@@ -11,5 +12,19 @@ internal sealed class KeyVaultSecretClientOptions : SecretClientOptions
 {
     public const string SectionName = "KeyVault";
 
-    public bool Enabled { get; set; }
+    // This value is based on the SecretClient's parameter name and is read automatically from the configuration
+    public Uri VaultUri { get; set; }
+
+    [Obsolete]
```

Comment:
Do we need to mark it obsolete? Should we just add comments for a future PR to cleanup as needed?

Reply:
I think I'll add a comment in addition to the `[Obsolete]` attribute. I like the attribute as it will prevent further dependency on it